importer/MtDe.py

```diff
@@ -0,0 +1,96 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class MtDe(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Use the 'gemeinde' field as
+        municipality.
+        """
+        municip_raw = self.gemeinde
+        municip_q = utils.q_from_wikipedia("de", municip_raw)
+        if municip_q:
+            self.add_statement("located_adm", municip_q)
+        else:
+            self.add_to_report("gemeinde", self.gemeinde, "located_adm")
+
+    def update_labels(self):
+        labels = {
+            "de": self.name_de,
+            "en": self.name_en,
+            "mt": self.name_mt
+        }
+        for language in labels:
+            self.add_label(language, labels[language])
+
+    def update_descriptions(self):
+        """
+        Set the descriptions in English.
+
+        Try to add the place name. Since the municipality
+        names are titles of articles, sometimes they
+        have the form of "$municipality (Malta)", in which
+        case remove the bracketed part.
+        """
+        placename = "Malta"
+        description = "cultural property in {}"
```

Comment:
I'll reach out and see if I can't get a description template for maltese.

Reply:
The maltese pattern would be `propjeta' kulturali ġewwa <municipality>, Malta`

Reply:
Done.